### ISSUE TITLE
Tests cannot run in parallel with pytest-xdist

### ISSUE THREAD

**Issue by Apteryks** [adobe-type-tools/afdko#1498]
Hello,

Currently the test suite cannot be made to run in parallel, because the temporary directory is shared between tests and that cause race conditions (FileNotFound errors and the likes).

It seems like it could be fixed simply by using a true temporary directory that expands to something unique for each test, so they can run independently at the same time.

Thanks!

**Comment by josh-hadley** [comment]
Agreed, this would be a nice improvement for the test suite. If you'd like to give it a try, we will happily review a PR that implements it!
